purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/PurchasesIntegrationTest.kt

```diff
@@ -43,52 +33,19 @@ class PurchasesIntegrationTest {
             Constants.productIdToPurchase != "PRODUCT_ID_TO_PURCHASE"
     }
 
-    private val testTimeout = 5.seconds
-    private val currentTimestamp = Date().time
-    private val testUserId = "android-integration-test-$currentTimestamp"
     private val entitlementsToVerify = Constants.activeEntitlementIdsToVerify
         .split(",")
         .map { it.trim() }
         .filter { it.isNotEmpty() }
-    private val proxyUrl = Constants.proxyUrl.takeIf { it != "NO_PROXY_URL" }
-
-    private lateinit var mockBillingAbstract: BillingAbstract
-
-    private var latestPurchasesUpdatedListener: BillingAbstract.PurchasesUpdatedListener? = null
-    private var latestStateListener: BillingAbstract.StateListener? = null
-
-    @get:Rule
-    var activityScenarioRule = activityScenarioRule<MainActivity>()
 
     @Before
     fun setup() {
-        latestPurchasesUpdatedListener = null
-        latestStateListener = null
-
-        onActivityReady {
-            clearAllSharedPreferences(it)
-
-            mockBillingAbstract = mockk<BillingAbstract>(relaxed = true).apply {
-                every { purchasesUpdatedListener = any() } answers { latestPurchasesUpdatedListener = firstArg() }
-                every { stateListener = any() } answers { latestStateListener = firstArg() }
-            }
-
-            proxyUrl?.let { urlString ->
-                Purchases.proxyURL = URL(urlString)
-            }
-
-            Purchases.configure(
-                PurchasesConfiguration.Builder(it, Constants.apiKey)
-                    .appUserID(testUserId)
-                    .build(),
-                mockBillingAbstract
-            )
-        }
+        setupTest()
```

Comment:
Ideally we should also migrate these tests to use a single `onActivityReady`. But will do that in a different PR.